src/main/java/bio/terra/cli/command/server/List.java

```diff
@@ -21,6 +22,7 @@ protected void execute() {
     java.util.List<Server> allPossibleServers = Server.list();
     formatOption.printReturnValue(
         allPossibleServers.stream()
```

Comment:
this pattern is common enough that you could make a generic function that takes the list, the comparator's argument, and the map method reference.

Reply:
Good idea. Doesn't save much, but it's used pretty frequently so in aggregate I think worth it.